{
    "type": "bundle",
    "id": "bundle--35708ecf-0013-435d-a60c-eb2e864c7134",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--d11fab02-e898-4faf-a201-6cb295a2c19b",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-04-15T00:41:41.112483Z",
            "modified": "2025-04-15T00:41:41.112483Z",
            "name": "CVE-2025-32908",
            "description": "A flaw was found in libsoup. The HTTP/2 server in libsoup may not fully validate the values of pseudo-headers :scheme, :authority, and :path, which may allow a user to cause a denial of service (DoS).",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-32908"
                }
            ]
        }
    ]
}